src/devices/Mcp25xxx/Register/IRegister.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mcp25xxx.Register
+{
+    public interface IRegister
```

Comment:
this awfully reminds pointer - wondering if this abstraction makes sense.

perhaps the abstraction should be:

```csharp
byte Read(Address address)
```

with different name

(haven't look at the whole code yet so this might not make sense)

Reply:
>byte Read(Address address)

There is a command (or instruction per datasheet) for reading the register value.  I basically have the same thing you mentioned in the Mcpxxx.cs file, but have not completed yet.

As for `Address GetAddress();` in IRegister, I could change this to a property instead of method.  For example...

```csharp
public Address GetAddress() => Address.Eflg;
// Change to...
public Address Address => Address.Eflg;
```

Thoughts?

Reply:
Either will do (I generally prefer properties but this is personal choice) - I was wondering if the interface is even needed but probably doesn't make sense to question without going deeper into the code :smile:

Reply:
I like the property approach, too.  I will update all in a future commit after reviewing and getting the remaining registers completed (hopefully by Friday).

I think it will make sense how the interface will work once I get into more of the commands/instructions.  Hopefully it won't take too much to change later if needed.